src/coreclr/vm/callstubgenerator.cpp

```diff
@@ -1054,6 +1265,17 @@ PCODE CallStubGenerator::GetFPRegRangeRoutine(int x1, int x2)
     return m_interpreterToNative ? FPRegsRoutines[index] : FPRegsStoreRoutines[index];
 }
 
+#ifdef TARGET_ARM64
+PCODE CallStubGenerator::GetFPReg128RangeRoutine(int x1, int x2)
+{
+#if LOG_COMPUTE_CALL_STUB
+    printf("GetFPReg128RangeRoutine %d %d\n", x1, x2);
+#endif
+    int index = x1 * NUM_FLOAT_ARGUMENT_REGISTERS + x2;
+    return m_interpreterToNative ? FPRegs128LoadRoutines[index] : FPRegs128StoreRoutines[index];
+}
```

Comment:
We should check somewhere - probably in this function - to make sure that we aren't returning 0, since a bunch of the slots in the routines arrays are 0. Otherwise we'll get a cryptic tailcall of 0 at runtime and it won't be obvious why, I think?

Reply:
We should never get the zero, as the x1 is always <= x2.